sidekick/lib/src/commands/init_command.dart

```diff
@@ -13,35 +14,35 @@ class InitCommand extends Command {
   @override
   String get name => 'init';
 
+  @override
+  String get invocation => super.invocation.replaceFirst(
+        '[arguments]',
+        '<projectRoot?>',
+      );
+
   InitCommand() {
     argParser.addOption(
       'cliName',
       abbr: 'n',
-      help: 'The name of the CLI to be created. \n'
-          'The `_cli` prefix will be defined automatically.',
+      help: 'The name of the CLI to be created (entryPoint name).',
```

Comment:
![](https://media.giphy.com/media/26tP4gFBQewkLnMv6/giphy.gif)

Wait, I remember that you proposed to use `entryPoint` instead of `entrypoint` and I now use it in all APIs like https://pub.dev/documentation/sidekick_core/latest/sidekick_core/SidekickContext/entryPoint.html

Reply:
Oh, you're right 😅 It just seemed weird to me to print `entryPoint` to the user, I would have expected `entrypoint` to be printed to the user. Just like I wouldn't print `cliName` to the user but `CLI name`. But in the end, it doesn't even matter as this is just a nitpick

Reply:
Btw, dart Isolate API also uses `entrypoint`, I think that's where I initially picked it up https://api.dart.dev/stable/2.19.0/dart-isolate/Isolate/spawn.html